backend/api/swagger.yml

```diff
@@ -513,6 +513,145 @@ components:
     Property:
       type: object
       properties:
-        id:
+        landlord_id:
+          type: string
+          format: uid
+          description: ID of landlord
+          example: 359627C2-C1E2-4CC3-89BD-3D33E9EC2AAB
+        voucher_type_accepted:
```

Comment:
@jkropko 

It may be a good idea long-term to add a building number field, which cross references to Charlottesville's open data building dataset